doc/man7/provider-cipher.pod

```diff
@@ -235,7 +235,11 @@ Gets the IV length for the associated cipher algorithm.
 
 =item B<OSSL_CIPHER_PARAM_IV> (octet_string OR octet_ptr)
 
-Gets the IV for the associated cipher ctx.
+Gets the current IV for the associated cipher ctx.
+
+=item B<OSSL_CIPHER_PARAM_ORIGINAL_IV> ("oiv") <octet_string OR octet_ptr>
```

Comment:
"oiv" isn't the best of names.  "original-iv" or "initial-iv" would be better, which one is a matter of debate...

Reply:
Also, we've changed the parameter naming convention to this:

```
=item "name" (B<OSSL_BLAH_PARAM_NAME>) <type spec>
```

Reply:
Removed this from the pod file - it now just uses OSSL_CIPHER_PARAM_IV